K8SSAND-1209 ⁃ Quick start installation fails if local Docker registry already present

Following [quick start instructions](https://github.com/k8ssandra/k8ssandra-operator/blob/main/docs/install/README.md) to install k8ssandra-operator on a local Kind cluster, I get the following after running `make single-up`:

```
docker: Error response from daemon: driver failed programming external connectivity on endpoint kind-registry (47267693e90c122b7f33facecb2637fadefdad508656626130b14a7a60ecbb5d): Bind for 0.0.0.0:5000 failed: port is already allocated.
make: *** [create-kind-cluster] Error 125
```

I had previously started a local Docker registry for some other task. I stopped my registry and ran make single-up again, just wondering if the docs should warn about this, or perhaps the makefile should check if there’s already a registry available and try to use it?



┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1209) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-1209
┆priority: Medium

--- a/docs/install/README.md
+++ b/docs/install/README.md
@@ -22,6 +22,8 @@ The examples in this guide use [kind](https://kind.sigs.k8s.io/) clusters. Insta
 
 By default kind clusters run on the same Docker network which means we will have routable pod IPs across clusters.
 
+**Note:**  Issues creating multiple kind clusters have been observed on various versions of Docker Desktop for Mac.  These issues seem to be resolved with the 4.5.0 release of Docker Desktop.  Please be sure to upgrade Docker Desktop if you plan to deploy using kind.
+
 ### **kubectx**
 
 [kubectx](https://github.com/ahmetb/kubectx) is a really handy tool when you are dealing with multiple clusters. The examples will use it so go ahead and install it now.
@@ -63,6 +65,8 @@ At completion, the cluster is now ready to accept a `K8ssandraCluster` deploymen
 
 **Note:** if a k8ssandra-0 and/or k8ssandra-1 kind cluster already exists, running `make 
 single-up` or `make multi-up` will delete and recreate them.
+
+**Note:** These steps will attempt to start a local Docker registry instance to be used by the kind cluster(s), if you are already running one locally it will need to be stopped before following these procedures.
 
 ### Single Cluster
 

